Configurations/10-main.conf

```diff
@@ -239,9 +239,8 @@ my %targets = (
         CFLAGS           => add_before(picker(default => "-Wall",
                                               debug   => "-O0 -g",
                                               release => "-O3 -fomit-frame-pointer")),
-        cflags           => add(threads("-pthread")),
+        cflags           => add("-m64"),
```

Comment:
Adding `-m64` to an x86 target?

Reply:
That was a mistake I've removed that now.